lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -789,6 +789,11 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
+        'path': '/name/0/suffix',
+        'op': 'replace',
+        'value': ['Jr.']
```

Comment:
Can the sequence be same as above, like, op, path and then value?

Reply:
Done here:https://github.com/cerner/fhir.cerner.com/pull/550/commits/4946d9bb50cbe875bd66237ff6bdf3a289268f11#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR794

Reply:
Done:https://github.com/cerner/fhir.cerner.com/pull/550/commits/4946d9bb50cbe875bd66237ff6bdf3a289268f11#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR794